src/twisted/conch/test/test_conch.py

```diff
@@ -343,20 +344,21 @@ def setUp(self):
             self.echoServerV6 = reactor.listenTCP(0, EchoFactory(), interface="::1")
             self.echoPortV6 = self.echoServerV6.getHost().port
 
-    def tearDown(self):
+    def tearDown(self) -> Any:
+        # c.f. https://github.com/twisted/twisted/issues/12417
```

Comment:
Just an explicit FIXME/TODO and the updated link to the non-duplicate issue

```suggestion
        # FIXME:https://github.com/twisted/twisted/issues/12037
```

For todos/fixmes we might want to use one of the format supported by  [flake8-todos](https://github.com/orsinium-labs/flake8-todos/)

This will make it easy to validate the format using `ruff`

Reply:
I very much want to avoid automated tooling doing anything with this comment; the documentation on this strikes me that it wasn't very thoughtful (there's no documentation, and the readme doesn't even suggest what it means to "check" a TODO) and so I'd prefer to avoid tripping some bogus false-positive busywork stuff that ruff likes to add in case we do switch to ruff.  There's also no "TODO" here until the other bug is fixed anyway.